tools/circt-reduce/circt-reduce.cpp

```diff
@@ -142,7 +148,18 @@ static LogicalResult writeOutput(ModuleOp module) {
         << outputFilename << "\": " << errorMessage << "\n";
     return failure();
   }
-  module.print(output->os());
+  if (emitBytecode) {
+    if (failed(writeBytecodeToFile(
+            module, output->os(),
+            mlir::BytecodeWriterConfig(getCirctVersion())))) {
+      mlir::emitError(UnknownLoc::get(module.getContext()),
+                      "failed to emit bytecode to file \"")
+          << outputFilename << "\n";
+      return failure();
+    }
+  } else
+    module.print(output->os());
```

Comment:
IIRC the coding guidelines want consistent use of braces, i.e., if the then-block has them the else-block should also have them.

Reply:
👍 This will be fixed in commit 32cc70e41.